.rubocop.yml

```diff
@@ -21,37 +22,22 @@ AllCops:
 Layout/CaseIndentation:
   Enabled: false
 
-Layout/EmptyLinesAroundAttributeAccessor:
-  Enabled: true
-
 Layout/FirstArrayElementIndentation:
   EnforcedStyle: consistent
 
 Layout/HashAlignment:
   Enabled: false
 
-Layout/LineLength:
-  Max: 120
-
 Layout/MultilineMethodCallIndentation:
   EnforcedStyle: indented
 
-Layout/SpaceAroundMethodCallOperator:
-  Enabled: true
-
 Lint/AmbiguousBlockAssociation:
   Enabled: false
 
-Lint/RaiseException:
-  Enabled: true
-
 Lint/ScriptPermission:
   Exclude:
     - "Rakefile"
 
-Lint/StructNewOverride:
-  Enabled: true
```

Comment:
Because of `NewCops: enable`, we no longer need to explicitly opt in to all these rules.